app/lib/video_utils.js

```diff
@@ -19,6 +19,17 @@ export const getTimecode = (videoPlayer, video, baseTimecode) => {
   return getTimecodesOffset(video, videoPlayer) + baseTimecode
 }
 
+/**
+ * Returns HD thumbnail URL if any, otherwise returns the default thumbnail URL.
+ */
+export const getHDThumbnailUrl = (video) => {
+  if (video.youtube_id) {
+    return `https://img.youtube.com/vi/${video.youtube_id}/hqdefault.jpg`
```

Comment:
Sorry @steelstyle my suggestion in the issue was misleading, I just found that there is a higher quality image resolution:
ie: https://img.youtube.com/vi/-NN89JhzOvk/maxresdefault.jpg

```suggestion
    return `https://img.youtube.com/vi/${video.youtube_id}/maxresdefault.jpg`
```

Reply:
No worries, I also did my own research before doing this, I looked at what Youtube was doing and they used hq quality.
I therefore suggest that we match their behavior and keep the code as is.

Reply:
If YouTube uses this for their Open graph tags then let's do the same indeed!